pypasta/LinuxMailCharacteristics.py

```diff
@@ -92,15 +91,15 @@
                  'linux-rdma@vger.kernel.org', 'linux-remoteproc@vger.kernel.org', 'linux-renesas-soc@vger.kernel.org',
                  'linux-rtc@vger.kernel.org', 'linux-s390@vger.kernel.org', 'linux-samsung-soc@vger.kernel.org',
                  'linux-scsi@vger.kernel.org', 'linux-sctp@vger.kernel.org', 'linux-security-module@vger.kernel.org',
-                 'linux-serial@vger.kernel.org',  'linux-sgx@vger.kernel.org', 'linux-sh@vger.kernel.org',
+                 'linux-serial@vger.kernel.org', 'linux-sgx@vger.kernel.org', 'linux-sh@vger.kernel.org',
                  'linux-sparse@vger.kernel.org', 'linux-spi@vger.kernel.org', 'linux-tegra@vger.kernel.org',
                  'linux-tip-commits@vger.kernel.org', 'linux-trace-devel@vger.kernel.org',
                  'linux-unionfs@vger.kernel.org', 'linux-usb@vger.kernel.org', 'linux-watchdog@vger.kernel.org',
                  'linux-wireless@vger.kernel.org', 'linux-wpan@vger.kernel.org', 'linux-xfs@vger.kernel.org',
                  'live-patching@vger.kernel.org', 'lvs-devel@vger.kernel.org', 'netdev@vger.kernel.org',
                  'netfilter-devel@vger.kernel.org', 'platform-driver-x86@vger.kernel.org',
                  'reiserfs-devel@vger.kernel.org', 'selinux@vger.kernel.org', 'sparclinux@vger.kernel.org',
-                 'stable@vger.kernel.org','target-devel@vger.kernel.org', 'util-linux@vger.kernel.org',
+                 'stable@vger.kernel.org', 'target-devel@vger.kernel.org', 'util-linux@vger.kernel.org',
```

Comment:
Yep, will merge this. By the way: This list ist derived from the public inboxes. Note to self: We added new mailing lists, so this list might be incomplete. Will have to check that.